hack/functest.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash -ex
+
+if [ -z "$KUBEVIRTCI_PATH" ]; then
+    # sets the path of the CI scripts
+    KUBEVIRTCI_PATH="$(
+        cd "$(dirname "$BASH_SOURCE[0]")/"
+        readlink -f ../kubevirtci/cluster-up
+    )"
+fi
+
+
+if [ -z "$KUBEVIRTCI_CONFIG_PATH" ]; then
+    # set location of ci configuration.
```

Comment:
What code does is usually easy enough to figure out, its the *why* that is important to document in situations like this (where you are defining a variable that based on reading the file appears to have no reason to exist).

Reply:
In